src/actions/ProfileActions.js

```diff
@@ -48,17 +37,25 @@ export const fetchProfileReset = () => ({
   type: FETCH_PROFILE.RESET,
 });
 
-export const fetchProfile = username => ({
-  type: FETCH_PROFILE.BASE,
-  payload: { username },
+// SAVE PROFILE ACTIONS
+
+export const saveProfile = formId => ({
```

Comment:
To save a profile, we just need to pass which form was being submitted.  The Saga can get the full data out of the reducer itself.